printer/opts.go

```diff
@@ -195,3 +200,47 @@ func WithUpgradeNotice(owner, repo string, opts ...upgrade.Options) *EnableUpgra
 func (c *EnableUpgradeNotice) ApplyToContainerOption(cfg *ContainerOptions) {
 	cfg.UpgradeNotice = upgrade.NewGitHubDetector(c.owner, c.repo, c.upgradeOpts...)
 }
+
+// WithPrettyStyleFromEnv Load a custom style from environment variable
+func WithPrettyStyleFromEnv(envVariable string) (CustomPrettyStyle, error) {
+	path := os.Getenv(envVariable)
+	options, err := parseConfigFile(path)
+
+	return options, err
+}
+
+// WithPrettyStyleFile Load a custom style from file
+func WithPrettyStyleFile(path string) (CustomPrettyStyle, error) {
+	options, err := parseConfigFile(path)
+
+	return options, err
+}
+
+func parseConfigFile(fileName string) (CustomPrettyStyle, error) {
+	var options CustomPrettyStyle
+	styleConfig := style.DefaultConfig(PrettyLayoutGoTpl)
+	if fileName == "" {
+		return options, nil
+	}
+
+	body, err := os.ReadFile(fileName)
+	if err != nil {
+		return options, err
+	}
+
+	extension := filepath.Ext(fileName)
+	switch extension {
+	case ".yml", ".yaml":
+		err = yaml.Unmarshal(body, &styleConfig)
+	case ".json":
+		err = json.Unmarshal(body, &styleConfig)
+	}
+
+	if err != nil {
+		return options, err
+	}
+	options = CustomPrettyStyle{
+		cfg: styleConfig,
+	}
+	return options, nil
+}
```

Comment:
after applying the rule to always return default value, it will be:
```suggestion
		err = yaml.Unmarshal(body, &options.cfg)
	case ".json":
		err = json.Unmarshal(body, &options.cfg)
	}

	return options, err
}
```